docs/kb/function-tutorial-intro.md

```diff
@@ -0,0 +1,239 @@
+---
+layout: docs
+toc_id: function-tutorial-intro
+title: VMware Event Broker Appliance - Function Tutorial Intro
+description: VMware Event Broker Appliance - Function Tutorial - Intro
+permalink: /kb/function-tutorial-intro
+cta:
+ title: What's next?
+ description: Continue function tutorial
+ actions:
+    - text: Modify and test a function locally on your workstation [Function Tutorial - Modify and Test](function-tutorial-modtest)
+    - text: Deploy a function to your VEBA appliance [Function Tutorial - Function Deploy](function-tutorial-deploy)
+---
+
+# In-depth Function Tutorial - Intro
+
+This tutorial will go over:
+- What is a VEBA function
+- The anatomy of a PowerShell function and how to do basic modifications to it.  We will use the [kn-ps-slack](https://github.com/vmware-samples/vcenter-event-broker-appliance/tree/master/examples/knative/powershell/kn-ps-slack) function for our example.
+- Setting up the required tools on your workstation: git, docker, and kubectl
+- Modify and test a function locally on your workstation
+- How to deploy the function to the Kubernetes cluster in your VEBA appliance
+
+## Table of Contents
+- [The big picture](#the-big-picture)
+- [The anatomy of a VEBA function](#the-anatomy-of-a-veba-function)
+- [Installing required tools on your workstation](#installing-required-tools-on-your-workstation)
+
+## The big picture
+VEBA functions provide the "custom" functionality to the VEBA appliance to fulfill your business requirements.  Functions are packaged as Docker images.  The functions can be written in PowerShell, Python, Go, or just about any language, but they are packaged and distributed as Docker images.  This is because the VEBA appliance runs Kubernetes (on top of the Photon OS) and functions are deployed as containers on the Kubernetes system.  Kubernetes provides an abstraction layer to allow containers to run seamlessly on disparate hardware/OSes.
```

Comment:
```suggestion
VEBA functions provide the "custom" logic to the VEBA appliance to fulfil your business requirements.  Functions are packaged as Docker images.  The functions can be written in PowerShell, Python, Go, or just about any language, and are packaged and distributed as Docker images.  This is because the VEBA appliance runs Kubernetes (on top of the Photon OS) and functions are deployed as containers on the Kubernetes system.  Kubernetes provides an abstraction layer to allow containers to run seamlessly on disparate hardware/OSes.
```

Reply:
Fixed.